src/proposer.ts

```diff
@@ -470,6 +471,28 @@ async function publishBundle(data: string, signature: string, from: string) {
 		logger.error('Failed to update balances:', error)
 		throw new Error('Balance update failed')
 	}
+	try {
+		const { WEBHOOK_URL, WEBHOOK_SECRET } = getEnvConfig()
+
+		if (WEBHOOK_URL && WEBHOOK_SECRET) {
+			const payload = {
+				type: 'BUNDLE_PROPOSED',
+				executions: cachedIntentions,
```

Comment:
We should rename this to `intentions`, which will require changes in a few places.

Reply:
But we don't need to do that in this PR, we can do it in a follow-up.